scalecube-organization/src/main/java/io/scalecube/organization/OrganizationServiceImpl.java

```diff
@@ -115,32 +123,40 @@ public static Builder builder() {
                 item.apiKeys(),
                 item.email(),
                 item.ownerId())).collect(Collectors.toList());
+        logger.debug("getUserOrganizationsMembership: exit, request: {}, return: {} memberships",
+            request, results.size());
         result.success(new GetMembershipResponse(
             infoItems.toArray(new OrganizationInfo[results.size()])));
       } catch (Throwable ex) {
+        logger.error("getUserOrganizationsMembership: error={}, request: {}", ex, request);
         result.error(ex);
       }
     });
   }
 
   @Override
   public Mono<DeleteOrganizationResponse> deleteOrganization(DeleteOrganizationRequest request) {
-
+    logger.debug("deleteOrganization: enter, request: {}", request);

```

Comment:
According to [Reactor 3 Reference Guide](http://projectreactor.io/docs/core/release/reference/) we should try to follow convention "nothing happens until subscribe".

So, to avoid logging anything if client's code doesn't subscribe, let's move this `logger.debug` inside `Mono.create`.